sound/soc/codecs/rt711-sdca.c

```diff
@@ -824,6 +896,8 @@ static const struct snd_kcontrol_new rt711_sdca_snd_controls[] = {
 		SDW_SDCA_CTL(FUNC_NUM_MIC_ARRAY, RT711_SDCA_ENT_PLATFORM_FU15, RT711_SDCA_CTL_FU_CH_GAIN, CH_R),
 		8, 3, 0,
 		rt711_sdca_set_gain_get, rt711_sdca_set_gain_put, mic_vol_tlv),
+	SOC_ENUM_EXT("GE49 Selected Mode", rt711_sdca_ge_mode_enum,
```

Comment:
Is "GE49 Selected Mode" how this will appear in the user interface? "Selected" and "Mode" are very vague and I'm afraid most users will have no idea what "GE49" means.

Can we have one or two more specific words like "Jack detection" or "microphone" or "headset"?

Reply:
In the SDCA context "GE Selected Mode" makes perfect sense.
It's been a while that users can't figure out what the controls mean...